frontend/src/metabase/nav/components/AdminNavbar/AdminNavbar.styled.tsx

```diff
@@ -27,6 +27,7 @@ export const AdminNavbarItems = styled.ul`
   display: flex;
   flex-grow: 0;
   min-width: 0;
+  gap: 4px;
```

Comment:
This is here so that when you hover on an item next to a selected one, they have a tiny bit of space between them.